src/legacy/ui/public/visualize/loader/pipeline_helpers/build_pipeline.test.js

```diff
@@ -170,15 +170,15 @@ describe('visualize loader pipeline helpers: build pipeline', () => {
       const params = {};
 
       it('without buckets', () => {
-        const schemas = { metric: [0] };
+        const schemas = { metric: [{ accessor: 0 }] };
         const actual = buildPipelineVisFunction.tagcloud({ params }, schemas);
         expect(actual).toMatchSnapshot();
       });
 
       it('with buckets', () => {
         const schemas = {
-          metric: [0],
-          segment: [1, 2]
+          metric: [{ accessor: 0 }],
+          segment: [{ accessor: 1 }]
```

Comment:
We should probably add all of the new params here so that the snapshots are testing everything

Reply:
i'll add another test for that, as this actually hit a bug :)